src/edu/psu/sweng500/team8/gui/CellGUI.java

```diff
@@ -119,7 +119,49 @@ public void populate(Cell cell, GameSession gameSession, boolean isRefresh,
 				this.numberInputField.setFocusable(true);
 
 			} else {
-				cellLostFocus(null);
+				/* Pencil Marks */
+				this.numberInputCell.setVisible(false);
+				this.pencilMarkDisplayCell.setVisible(true);
+				
+				for (int i = 1; i <= 9; i++) {
+					
+					int rowIndex = ((i - 1) / 3);
+					int columnIndex = ((i - 1) % 3);
+
+					this.pencilMarkDisplay[rowIndex][columnIndex]
+							.setVisible(true);
+					this.pencilMarkDisplay[rowIndex][columnIndex]

```

Comment:
1. we still need mouse adapter as we are still clicking pencilMarkDisplayCell to select the cell. 
2. updating code to add mouse adapter once.